spec/views/form_drafts/edit_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'rails_helper'
+include RSpecHtmlMatchers
+
+describe 'form_drafts/edit.haml' do
+  before :each do
+    @draft = create :form_draft
+    @top_field = create :field, form_draft: @draft
```

Comment:
I'd prefer to align the `=` of lines 7-9, to make it obvious that they're all the same invocation.